custodia/httpd/server.py

```diff
@@ -436,25 +464,39 @@ class HTTPServer(object):
 
     def __init__(self, srvurl, config):
         url = urlparse(srvurl)
-        address = unquote(url.netloc)
         if url.scheme == 'http+unix':
             # Unix socket
+            address = None
+            if sd_listen_fds is not None:
```

Comment:
Python doesn't expose a generic getsockopt() function and https://docs.python.org/3/library/socket.html#socket.fromfd requires the caller to supply the correct af family and sock type. Socket activation on a TCP port is not as useful as AF_UNIX for SSSD responder.